app/assets/javascripts/components/containers/mastodon.jsx

```diff
@@ -50,12 +50,14 @@ import es from 'react-intl/locale-data/es';
 import fa from 'react-intl/locale-data/fa';
 import fi from 'react-intl/locale-data/fi';
 import fr from 'react-intl/locale-data/fr';
+import hr from 'react-intl/locale-data/hr';
 import hu from 'react-intl/locale-data/hu';
 import it from 'react-intl/locale-data/it';
 import ja from 'react-intl/locale-data/ja';
 import pt from 'react-intl/locale-data/pt';
 import nl from 'react-intl/locale-data/nl';
 import no from 'react-intl/locale-data/no';
+import oc from 'react-intl/locale-data/oc';
```

Comment:
I seen [`locale-data` directory](https://unpkg.com/react-intl@2.1.5/locale-data/) for [`react-intl`](https://github.com/yahoo/react-intl). `oc.js` does not seem to exist.